src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -314,7 +314,13 @@ private void walk(TypedVar symbol, boolean isDefault) {
           // https://github.com/Microsoft/TypeScript/issues/3194
           emit("interface");
           emit(getUnqualifiedName(symbol));
-          visitObjectType(ftype, ftype.getPrototype());
+          visitObjectType(ftype, ftype.getPrototype(), false);
+
+          emit("var");
```

Comment:
Good catch. I started wondering how test pass in that case and discovered we were forking on isDefault unnecessarily.